js/app.js

```diff
@@ -15,6 +15,7 @@ let leftProductImage = document.getElementById('left_product_img');
 let centerProductImage = document.getElementById('center_product_img');
 let rightProductImage = document.getElementById('right_product_img');
 
+//do we need to declare these here?
 let leftProductOnThePage;
 let centerProductOnThePage;
 let rightProductOnThePage;
```

Comment:
I am not able to comment on your recent code because they don't have branches, so I'm gonna leave this here!

Under this line of code: let chartResults = document.getElementById('chartResults');

Add this underneath:
let viewResultsButton = document.querySelector('button');

That way your button works and is initialized